presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSecurityConfigProvider.java

```diff
@@ -0,0 +1,263 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import org.apache.kafka.common.config.SslConfigs;
+
+import java.util.Properties;
+
+/**
+ *  {@KafkaSecurityConfig} manages Kafka configuration related to connection security. I.e. if the security protocol
+ *  'SSL' is being used. If so, additional properties might be supplied, i.e. location and password of the truststore
+ *  and keystore to be used.
+ */
+public class KafkaSecurityConfigProvider
+{
+    private Properties props;
+
+    private String sslProtocol;
+    private String sslProvider;
+    private String sslCipherSuites;
+    private String sslEnabledProtocols;
+    private String sslKeystoreType;
+    private String sslKeystoreLocation;
+    private String sslKeystorePassword;
+    private String sslKeyPassword;
+    private String sslTruststoreType;
+    private String sslTruststoreLocation;
+    private String sslTruststorePassword;
+    private String sslKeymanagerAlgorithm;
+    private String sslTrustmanagerAlgorithm;
+    private String sslEndpointIdentificationAlgorithm;
+    private String sslSecureRandomImplementation;
+
+    public KafkaSecurityConfigProvider()
```

Comment:
drop

Reply:
? Previous review remarks wanted to separate the security related stuff, therfore i moved it out of "KafkaConfig" - should i move the 5 remaining (most used) properties back into KafkaConfig directly?

Reply:
No. Here I meant to drop empty constructor, parameterless constructor. It is not needed :)

Reply:
yes, should have properly checked the line you commented on :D thought this was on `public class KafakSecurityConfig`....